x-pack/plugins/security_solution/public/common/components/event_details/overview/overview_card.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
+import React from 'react';
+
+import { ActionCell } from '../table/action_cell';
+import { euiStyled } from '../../../../../../../../src/plugins/kibana_react/common';
+import { EnrichedFieldInfo } from '../types';
+
+const ActionWrapper = euiStyled.div`
+  width: 0;
+  transform: translate(6px);
+  transition: transform 50ms ease-in-out;
+  margin-left: ${({ theme }) => theme.eui.paddingSizes.s};
+`;
+
+const OverviewPanel = euiStyled(EuiPanel)`
+  &&& {
```

Comment:
TIL. Had never seen `&&&` before.

Reply:
Also, not to come in last minute here. Just wanted to make sure that we didn't want to use the EUICard for this: https://elastic.github.io/eui/#/display/card#custom-children

Reply:
Haha, yeah, `&&&` is fun and it looks a lot less like a hack than `.myClass.myClass.myClass` even though it's exactly the same :D 

Regarding using `EUICard`: I looked into it in the beginning but I would have ha to override a lot of styles and then decided to go with the custom components.

Reply:
Sounds good. Thanks!